packages/nx-plugin/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.29.0",
   "license": "MIT",
   "dependencies": {
-    "@nx/devkit": "^17.1.3",
+    "@nx/devkit": "^17.0.0 || ^18.0.0",
```

Comment:
would a ">" operator also work to include all 2 versions?

Reply:
yes, it would accept all versions above X. It might be too permissive if a breaking change occurs in the later (Nx) major version.

Reply:
ok cool! thx for the answer! the current implementation is good!